ark/phenotyping/som_utils.py

```diff
@@ -51,40 +49,6 @@ def normalize_rows(pixel_data, channels):
     return pixel_data_sub
 
```

Comment:
In the description for the normalize_rows function, can remove the part that says "and removes rows that sum to 0" since you are now doing that outside of this function